File: csharp/UnitTest/Security/TextRisk/ClientTest.cs
using NUnit.Framework;
using Alipay.EasySDK.Factory;
using Alipay.EasySDK.Security.TextRisk.Models;

namespace UnitTest.Security.TextRisk
{
    public class ClientTest
    {
        [SetUp]
        public void SetUp()
        {
            Factory.SetOptions(TestAccount.Mini.CONFIG);
        }

        [Test]
        public void TestDetect()
        {
            AlipaySecurityRiskContentDetectResponse response = Factory.Security.TextRisk().Detect("test");

            Assert.AreEqual(response.Code, "10000");
            Assert.AreEqual(response.Msg, "Success");
            Assert.Null(response.SubCode);
            Assert.Null(response.SubMsg);
            Assert.NotNull(response.HttpBody);
            Assert.AreEqual(response.Action, "PASSED");
            Assert.NotNull(response.UniqueId);
        }
    }
}

File: php/test/security/textrisk/ClientTest.php
<?php


namespace Alipay\EasySDK\Test\security\textrisk;


use Alipay\EasySDK\Kernel\Factory;
use Alipay\EasySDK\Test\TestAccount;
use PHPUnit\Framework\TestCase;

class ClientTest extends TestCase
{
    public function testDetectContent(){
        $account = new TestAccount();
        $app = new Factory($account->getTestAccount());
        $result = $app->security()->textRisk()->detect("test");
        $this->assertEquals('10000', $result['code']);
        $this->assertEquals('Success', $result['msg']);
    }
}